File: app/src/main/java/org/fossasia/openevent/general/model/LoginResponse.java
package org.fossasia.openevent.general.model;

import com.google.gson.annotations.SerializedName;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by nikit on 11/5/18.
 */
@Data
@EqualsAndHashCode(callSuper = false)
public class LoginResponse {
    @SerializedName("access_token")
    private String accessToken;
    public String getAccessToken() {
        return accessToken;
    }
}

File: app/src/main/java/org/fossasia/openevent/general/model/LoginResponse.kt
package org.fossasia.openevent.general.model

import com.fasterxml.jackson.databind.PropertyNamingStrategy
import com.fasterxml.jackson.databind.annotation.JsonNaming


@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
data class LoginResponse(var response: Int? = null, var accessToken: String? = null)